pkg/vulnsrc/osv/osv.go

```diff
@@ -0,0 +1,193 @@
+package osv
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"path/filepath"
+	"time"
+
+	bolt "go.etcd.io/bbolt"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy-db/pkg/db"
+	"github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy-db/pkg/utils"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/vulnerability"
+	vtypes "github.com/aquasecurity/vuln-list-update/osv"
+)
+
+const (
+	osvDir         = "osv"
+	platformFormat = "%s::Osv Security Advisories"
```

Comment:
You can this function.
https://github.com/aquasecurity/trivy-db/blob/3c60651dc9c4195e57e7057b853a7f37859bf0fb/pkg/vulnsrc/bucket/bucket.go#L16

Reply:
used this function